docker-compose.dev.yml

```diff
@@ -22,12 +25,13 @@ services:
     volumes:
       - ckan_storage:/var/lib/ckan
       - ./src:/srv/app/src_extensions
-      - /usr/lib/python3.10/site-packages
-      - /root/.vscode-server
+      - pip_cache:/root/.cache/pip
+      - site_packages:/usr/lib/python3.10/site-packages
```

Comment:
Is this actually going to work?

This isn't going to be live at build time, so your install is going to go into the image. Presumably, this will initialize the volume with the contents of the directory when first run, but then if you rebuild the image, the site-packages are going to be out of date.

Reply:
The startup script re-installs everything so if you have an out of date volume locally and upgrade ckan versions it should only dowload and re-install once.